snap/snapcraft.yaml

```diff
@@ -950,7 +960,7 @@ parts:
 
   kuiper:
     source: https://github.com/emqx/kuiper.git
-    source-tag: 0.4.1
+    source-tag: 0.9.1
```

Comment:
Please change to version 1.0.0, which is now released and what we want for code freeze. I will be updating the compose files for this version today.

Reply:
Updated to 1.0.0